xonsh/xontribs.json

```diff
@@ -71,6 +71,11 @@
   "url": "https://github.com/Granitas/xonsh-vox-tabcomplete",
   "description": ["Adds tabcomplete functionality to vox inside of xonsh."]
  },
+ {"name": "click_tabcomplete",
+  "package": "xonsh-click-tabcomplete",
+  "url": "https://github.com/Granitas/xonsh-click-tabcomplete",
```

Comment:
Did you just changed the usernmame of github? also the url of `xonsh-vox-tabcomplete`.